numpyro/infer/util.py

```diff
@@ -151,7 +151,7 @@ def potential_energy(model, model_args, model_kwargs, params):
 
 def _init_to_unconstrained_value(site=None, values={}):
     if site is None:
-        return partial(init_to_value, values=values)
+        return partial(_init_to_unconstrained_value, values=values)
```

Comment:
@neerajprad I think this is my original intention. My last fix is a bit misleading (though it still solves the issue)... :(

Reply:
Thanks, now I understand what's happening. :)